bluecherrydvr/unity
# Correctly dispose current players on close

Fixes #49 
Fixes #53

To test:
 1. Compile, Install and run the app;
 2. Wait a few moments and close the app;
 3. You should see the app in the task manager while the resources are being disposed;
 4. App should the fully closed
